linode/manager
# style: [M3-6215] - Metadata final copy

## Description 📝
- Update Metadata features with final UX copy
    - Doc links are still TBD and will be addressed in a separate ticket
- Fix help icon styling

## Preview 📷
![image](https://user-images.githubusercontent.com/115299789/230648811-b0dd492a-6ee8-47ba-873e-edcabaf6cd94.png)

## How to test 🧪
Turn on the MSW, and check the following flows for updated copy based off UX docs:
- Image Capture/Image Upload
- Linode Rebuild
- Create from Distro/Backups/Clone
    - Select a metadata-test image and London region
